### PR TITLE
Update min app for qa release

### PR DESCRIPTION
**Pull request by duclong201**
Update for qa release
- bugs fix

### CODE REVIEW

**Comment by duclong201** [comment]
@datkinnguyen , @tyc222 , Please review this when you have time. Thanks

File: lib/tempo_page.dart [hunk 1]
```diff
@@ -88,63 +101,74 @@ class _TempoPageState extends State<TempoPage> {
     tempoEventChannel.setMethodCallHandler((MethodCall call) async {
       var args = call.arguments;
       switch (call.method) {
-        case TempoEvents.tempoTrackingDidStopWithError:
-          debugPrint("TempoTrackingStoppedWithError: $args");
+        // TODO: change name to dart constant when plugin is merged to release
```
Comment (datkinnguyen) [inline lib/tempo_page.dart:104]: leftover TODO:

File: lib/tempo_page.dart [hunk 2]
```diff
@@ -88,63 +101,74 @@ class _TempoPageState extends State<TempoPage> {
...
+        case TempoEvents.tempoTrackingStoppedWithError:
+          var errorCode = args["code"];
+          var errorMessage = args["message"];
+          showError("Tempo Tracking Stopped With Error", "$errorCode $errorMessage", context);
```
Comment (datkinnguyen) [inline lib/tempo_page.dart:108]: single quote?

File: lib/tempo_page.dart (hunk 1 above)
Comment (duclong201) [reply lib/tempo_page.dart]: I have updated this

File: lib/initial_page.dart [hunk 3]
```diff
@@ -34,6 +36,17 @@ class _InitialPageState extends State<InitialPage> {
     }
   }
 
+  void _clearSharedPreferences() async {
+    final sharedPrefs = await SharedPreferences.getInstance();
+    await sharedPrefs.remove('destinationId');
+    await sharedPrefs.remove('projectId');
+  }
```
Comment (datkinnguyen) [inline lib/initial_page.dart:43]: this method is duplicated in another place, would be better if you could move it to a helper/utility class?

**Review by datkinnguyen — APPROVED** [review]
Looking good 👍

**Pull request merged**